app/uk/gov/hmrc/mtdsatestsupportapi/controllers/requestParsers/validators/CreateAmendITSAStatusValidator.scala

```diff
@@ -69,6 +61,22 @@ class CreateAmendITSAStatusValidator extends Validator[CreateAmendITSAStatusRawD
 
   }
 
+  private def submissionDatesUniquenessValidation(error: => MtdError = SubmittedOnFormatError): CreateAmendITSAStatusRawData => List[List[MtdError]] =
```

Comment:
Should this method argument be call-by-name? Probably error values are always constant and no lazy evaluation needed.

Reply:
Upon modifying the method argument to call-by-value (removing the fat arrow =>), as seen in the updated code snippet below:
`private def submissionDatesUniquenessValidation(error: MtdError = SubmittedOnFormatError)`

I ran the tests, and the code appears to work as expected. Let's discuss this further on a call to ensure we're aligned.